scipy/special/_ufuncs.pyx

```diff
@@ -6538,7 +6538,11 @@ cdef char ufunc_gamma_types[8]
 cdef char *ufunc_gamma_doc = (
     "gamma(z)\n"
     "\n"
-    "Gamma function.\n"
+    "Gamma function\n"
+    "\n"
+    ".. math::\n"
+    "\n"
+    "    \Gamma(z) = \int_0^\infty x^{z-1} e^{-x} dx = (z - 1)!\n"
```

Comment:
These files shouldn't be edited manually, instead add the formula to `add_newdocs.py` (this formula is missing from there), and run `generate_ufuncs.py`.